lib/_http_outgoing.js

```diff
@@ -120,6 +121,24 @@ function OutgoingMessage() {
 }
 util.inherits(OutgoingMessage, Stream);
 
+Object.defineProperty(OutgoingMessage.prototype, '_writableState', {
+  get() {
+    var state = new Writable.WritableState();
+    Object.defineProperty(this, '_writableState', {
+      value: state,
+      configurable: false,
+      enumerable: false
+    });
+    return state;
+  },
+  enumerable: false
+});
```

Comment:
There isn't any. Streams would not really work in the first place if that is overwritten.